src/ResourceManager/KeyVault/AzureRM.KeyVault.psd1

```diff
@@ -9,7 +9,7 @@
 @{  
   
 # Version number of this module.  
-ModuleVersion = '1.1.0'
```

Comment:
Should be 1.1.1, and should change the AssemblyFileVersion and AssemblyVersion in Properties\AssemblyInfo.cs to match